src/Omega_h_vector.hpp

```diff
@@ -150,6 +150,16 @@ OMEGA_H_INLINE Vector<n> zero_vector() OMEGA_H_NOEXCEPT {
   return fill_vector<n>(0.0);
 }
 
+template <Int n>
+OMEGA_H_INLINE Vector<n> max_vector() OMEGA_H_NOEXCEPT {
+  return fill_vector<n>(Kokkos::Experimental::finite_max_v<Real>);
```

Comment:
I missed that this was an `Experimental` function in my first pass.  Do you know if this exist in Kokkos 3.7?  That is currently the oldest version Omega_h supports:

https://github.com/SCOREC/omega_h/blob/0dcdfee7569200fd358716e1d06de982f30c9186/CMakeLists.txt#L59

Reply:
According to Kokkos documentation, [it was added in Kokkos 3.6](https://kokkos.org/kokkos-core-wiki/API/core/numerics/numeric-traits.html#:~:text=trait*%20denotes%20traits%20that%20were%20added%20in%20Kokkos%203.6) and no mention of change yet.

Reply:
OK.  Thank you.